ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -218,48 +222,72 @@
-    ))}]})
+ (let [metadata (json/parse-string (:metadata subscription))
+  concept-id (get-in subscription [:extra-fields :collection-concept-id])
+  meta-query (get metadata "Query")
+  sub-start-time (:start-time subscription)]
+  {:from from-email-address
+   :to to-email-address
+   :subject "Email Subscription Notification"
+   :body [{:type "text/html"
+    :content (markdown/md-to-html-string (str
+     "You have subscribed to receive notifications when data is added to the following query:\n\n"
+     "`" concept-id "`\n\n"
+     "`" meta-query "`\n\n"
+     "Since this query was last run at "
+     sub-start-time
+     ", the following granules have been added or updated:\n\n"
+     (email-granule-url-list gran-ref-location)
+     "\n\nTo unsubscribe from these notifications, or if you have any questions, please contact us at [cmr-support@earthdata.nasa.gov](mailto:cmr-support@earthdata.nasa.gov).\n"
+     ))}]}))
+
+(defn- add-updated-since
+ "Pull out the start and end times from a time-constraint value and associate them to a map"
+ [raw time-constraint]
+ (let [parts (clojure.string/split time-constraint, #",")
+  start-time (first parts)
+  end-time (last parts)]
+  (assoc raw :start-time start-time :end-time end-time)))
 
 (defn- process-subscriptions
   "Process each subscription in subscriptions."
   [context subscriptions time-constraint]
-  (doseq [subscription subscriptions
-         :let [email-address (get-in subscription [:extra-fields :email-address])
+  (error "* * * * * *")
+  (error "process-subscriptions" time-constraint)
+  (doseq [raw_subscription subscriptions
+         :let [subscription (add-updated-since raw_subscription time-constraint)
+               email-address (get-in subscription [:extra-fields :email-address])
                sub-name (get-in subscription [:extra-fields :subscription-name])
                coll-id (get-in subscription [:extra-fields :collection-concept-id])
                query-string (-> (:metadata subscription)
                                 (json/decode true)
                                 :Query)
                query-params (create-query-params query-string)
                params1 (merge {:created-at time-constraint
-                               :collection-concept-id coll-id}
+                               :collection-concept-id coll-id
+                               :token (cmr.transmit.config/echo-system-token)}
                               query-params)
                params2 (merge {:revision-date time-constraint
-                               :collection-concept-id coll-id}
+                               :collection-concept-id coll-id
+                               :token (cmr.transmit.config/echo-system-token)}
                               query-params)]]
-      (info "Processing subscription: " sub-name)
+      ; remove these comments before going to production
+      (error "********************************")
```

Comment:
Remove this error message and comment

Reply:
we put those in to make the logs clear while in sit, but I can remove them now.